test/libsolidity/gasTests/abiv2.sol

```diff
@@ -14,9 +14,9 @@ contract C {
 }
 // ----
 // creation:
-//   codeDepositCost: 1120000
-//   executionCost: 1160
-//   totalCost: 1121160
+//   codeDepositCost: 1094400
+//   executionCost: 1134
+//   totalCost: 1095534
```

Comment:
My guess here is that we used to generate pointer and non-pointer versions of the same functions and those become unified with this change...

Reply:
Or we did superfluous conversions between pointer and non-pointer types or something like that...

Reply:
https://gist.github.com/ekpyron/41f43961838274ad5f5dfb99c4b387e4
Looks identical short of function names... not sure why this changes...